### PR TITLE
::backdrop is well supported

### LINKED ISSUE #33957: HTML Demo for ::backdrop pseudo-element has invisible button
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/::backdrop

### What specific section or headline is this issue about?

HTML Demo for ::backdrop pseudo-element

### What information was incorrect, unhelpful, or incomplete?

In the HTML Demo, the button to show the dialog is invisible.

### What did you expect to see?

I expected the button to be visible.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/_doublecolon_backdrop`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/::backdrop
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/_doublecolon_backdrop/index.md
* Last commit: https://github.com/mdn/content/commit/1c4eb0bfb5f72a26fcc21a83fac91aa3e66c2fb8
* Document last modified: 2024-01-02T16:00:45.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
the previous example was a screen shot. fixes #33957

### CODE REVIEW

File: files/en-us/web/css/_doublecolon_backdrop/index.md [hunk 1]
```diff
@@ -42,21 +42,72 @@ Elements are placed in a last-in/first out (LIFO) stack in the top layer. The `:
...
+closeButton.addEventListener("click", () => {
+  dialog.close();
+});
+```
 
-![An almost full-screen video player with purple above and below the player as the video player doesn't completely fill the screen.](bbb-backdrop.png)
+#### Results
 
-[See this example in action](https://mdn.github.io/css-examples/backdrop/index.html), after changing the color of the background cause the video to go fullscreen to see the change to the backdrop color.
+{{EmbedLiveSample('Styling a modal dialog's backdrop', 450, 120)}}
```
Comment (estelle) [inline files/en-us/web/css/_doublecolon_backdrop/index.md:110]:
```suggestion
{{EmbedLiveSample("Styling a modal dialog's backdrop", 450, 120)}}
```

File: files/en-us/web/css/_doublecolon_backdrop/index.md [hunk 2]
```diff
@@ -42,21 +42,73 @@ Elements are placed in a last-in/first out (LIFO) stack in the top layer. The `:
 
 ## Examples
 
-### Styling the backdrop for fullscreen video
+### Styling a modal dialog's backdrop
 
-In this example, the backdrop style used when a video is shifted to fullscreen mode is configured to be a grey-blue color rather than the black it defaults to in most browsers.
+In this example, we style the backdrop used when a modal {{htmlelement("dialog")}} is opened using the `::backdrop` pseudo-element
```
Comment (wbamberg) [inline files/en-us/web/css/_doublecolon_backdrop/index.md:47]:
```suggestion
In this example, we use the `::backdrop` pseudo-element to style the backdrop that is used when a modal {{htmlelement("dialog")}} is open.
```

(the previous version was ambiguous, you could read it as: we use the :backdrop pseudo-element to open the dialog)

**Review by wbamberg — COMMENTED** [review]
Just one comment, but while I think this is overall a good change I don't think it fixes https://github.com/mdn/content/issues/33957, does it? It seems unrelated to the person's issue.

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Pull request merged**